src/portal/views/Dao.vue

```diff
@@ -365,6 +369,7 @@ export default class DaoView extends Vue {
                 this.$toasted.show("Voted for proposal!");
                 this.loadingVote = false;
                 this.openVDialog = false;
+                this.loadingProposals = true;
```

Comment:
I don't see why you'd include this here as the loading proposals isn't displayed when you are voting